plugins/modules/ipasudorule.py

```diff
@@ -552,6 +568,12 @@ def main():
                             + res_find.get('ipasudorunasextuser', [])
                         )
                     )
+                    runasuser_group_add, runasuser_group_del = (
+                        gen_add_del_lists(
+                            runasuser_group,
+                            res_find.get('ipasudorunas_group', [])
```

Comment:
There is no `list(res_find...)` in this call, but in the others below.

Reply:
Fixed by removing unnedded 'list()' conversions.